src/eel2_adapter.cpp

```diff
@@ -3,14 +3,17 @@
 #include "ns-eel-addfuncs.h"
 #include "ns-eel-int.h"
 
+#include <SC_InterfaceTable.h>
 #include <SC_Unit.h>
 
+static InterfaceTable *ft;
```

Comment:
`ft` is never initialized. Two solutions:

1. add a function that initializes `ft` and call it in `PluginLoad`.

2. *declare* `ft` as `extern` both in `eel2_adapter.cpp` and `dyngen.cpp`, but only *define* it in `dyngen.cpp`

Reply:
Add 2.:

In `eel2_adapter.cpp`:
```
extern InterfaceTable *ft;
```

In `dyngen.cpp`:
```
InterfaceTable *ft;
```